atom/common/crash_reporter/crash_reporter_linux.h

```diff
@@ -54,6 +54,7 @@ class CrashReporterLinux : public CrashReporter {
   uint64_t process_start_time_;
   pid_t pid_;
   std::string upload_url_;
+  bool upload_to_server_;
```

Comment:
Can we support `setUploadToServer` on Linux now as well that would update this value so it could be dynamically set like on macOS?

https://github.com/electron/electron/blob/master/docs/api/crash-reporter.md#crashreportersetuploadtoserveruploadtoserver-macos